src/main.jsx

```diff
@@ -1,142 +1,97 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
-import Login from "./views/LoginPage/Login";
-import { Routines } from "./views/Routines/Routines";
 
-import RegisterUser from "./views/RegisterPage/RegisterUser";
+import Login from "./views/LoginPage/Login";
 import RegisterClient from "./views/RegisterPage/RegisterClient";
-import GymMachineForm from "./views/MachineList/AddMachine";
-import "./index.css";
-import "@radix-ui/themes/styles.css";
-import { Theme } from "@radix-ui/themes";
 import App from "./App";
-import Users from "./views/UserListing/Users";
-import "@radix-ui/themes/styles.css";
-import MachineList from "./views/MachineList/MachineList";
 import MainLayout from "./views/MainLayout/MainLayout";
-import { EditRoutine } from "./views/Routines/EditRoutine";
-import { AuthProvider } from "./utils/context/AuthContext";
-import OwnerRoute from "./components/OwnerRoute";
+import { Theme } from "@radix-ui/themes";
+import "@radix-ui/themes/styles.css";
+import "./index.css";
```

Comment:
Resulta que el estilo por defecto de tailwind se está aplicando con preferencia al estilo por defecto de Radix, de ahí el error. Con cambiar el orden de los imports, se arregla el error.
```suggestion
import "./index.css";
import "@radix-ui/themes/styles.css";
```